client/my-sites/themes/controller.js

```diff
@@ -130,7 +131,16 @@ export function details( context, next ) {
 		isFullScreen: true
 	} ) );
 
-	context.primary = makeElement( ThemeSheetComponent, Head, context.store, props );
 	context.secondary = null; // When we're logged in, we need to remove the sidebar.
+	//TODO: use makeElement()
+	context.primary = (
+		<ReduxProvider store={ context.store } >
+			<Head title={ props.title } isSheet>
```

Comment:
Maybe this stuff, with some modifications? -> https://github.com/Automattic/wp-calypso/pull/3425/files#diff-85f90b61d9c36634c5464a3180470726R70